### DISCUSSION TITLE
TypeError: __init__() got an unexpected keyword argument 'weight'

### DISCUSSION THREAD

**Discussion by NastaranVB** [Project-MONAI/MONAI#7503] [Q&A]
I want to apply weight to the voxels of each class in DiceLoss, but I get the error " TypeError: __init__() got an unexpected keyword argument 'weight'" while using 
loss_function=DiceLoss(include_background=False, to_onehot_y=2, softmax=True, smooth_nr=0, smooth_dr=1e-5, weight = [1, 10]).

**Comment by KumoLiu** [comment]
Hi @NastaranVB, it's a new argument introduced after 1.3.
Perhaps try latest docker:
`docker pull projectmonai/monai:latest`
Or weekly version:
`pip install monai-weekly`

Hope it helps, thanks.

**Reply by NastaranVB** [reply]
@KumoLiu thanks for coming back to me so quickly. 
It is now working, Thanks.
